berserk/clients/account.py

```diff
@@ -50,6 +60,7 @@ def set_kid_mode(self, value: bool):
         params = {"v": value}
         self._r.post(path, params=params)
 
+    @deprecated("Use endpoint from the Bot client instead", version="0.12.8")
```

Comment:
I don't see a reason for this method to be deprecated, since when used it's still a client

Reply:
Well, in the official API the method is in Bot not in Account.